kubeflow/instance/gcp_config/kustomization.yaml

```diff
@@ -3,11 +3,20 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 # namespace should match the project.
-# This assumes we are running CNRM in namespace mode and namespaces match project names.
-namespace: project-id # {"$kpt-set":"project"}
-# TODO(jlewi): do not commit the labels auto-deploy and purpose; they were added
-# as part of autodeployment testing.
+# This assumes we are running a config connector in namespace mode and
+# namespaces match project names.
+namespace: PROJECT # {"$kpt-set":"project"}
 commonLabels:
   kf-name: KUBEFLOW-NAME # {"$kpt-set":"name-label"}
 resources:
+# The default Google Cloud resources.
 - ../../upstream/manifests/gcp/v2/cnrm
+# Add additional Google Cloud resources by including resources here.
+# For example, uncomment the following line to include an additional example
+# nodepool described in a local file ./nodepool-example.yaml.
+# - nodepool-example.yaml
```

Comment:
(For my education) This feels like some sort of patching as well. What if the resource specified in this yaml already exists? Will it be replaced? merged? or there will be some error thrown?

Reply:
If the resource already exists, then this will fail. User needs to switch to patchesStrategicMerge. Therefore, the intent is clear, if we include a resource, it's something new. If we add a patch, the base resource must exist.

Reply:
Thanks for the explanation!